### ISSUE TITLE
AppCenterUploaderException 1.3.0

### ISSUE THREAD

**Issue by Taost** [oliviergauthier/gradle-appcenter-plugin#81]
We are seeing intermittent failures to upload APKs to appcenter. Recently it happens much more often. It seems the failure falls to this line of the plugin. https://github.com/oliviergauthier/gradle-appcenter-plugin/blob/master/src/main/kotlin/com/betomorrow/gradle/appcenter/infra/AppCenterUploader.kt#L74

```
* What went wrong:
11:01:37  Execution failed for task ':xxx:appCenterUploadApkxxx'.
11:01:37  > com.betomorrow.gradle.appcenter.infra.AppCenterUploaderException: Fetching release id failed: Tried 60 times.
```

We are running plugin version 1.3.0. Please let us know if you have any suggestion. Thanks!

**Comment by Rolf-Smit** [comment]
I see the same thing, it happens way more often. The https://status.appcenter.ms/ page shows no issues, however I'm pretty sure this is an AppCenter server issue. Lately I have grown more and more unsatisfied with the service, due to issues like this.

TLDR: I doubt something can be done about this in this Gradle Plugin.

**Comment by Rolf-Smit** [comment]
Got me thinking, should we increase the timeout between trying to get the upload? Maybe 1 second times 60, is just one minute, maybe that is too fast?

**Comment by Rolf-Smit** [comment]
@Taost my Pull-request is based on the official AppCenter CLI,  which also works without a limit.

However waiting forever felt a bit extreme. Instead it waits for 15 minutes (Instead of 60 seconds or forever), I feel like that is more than enough for a CI task.

**Comment by Rolf-Smit** [comment]
@oliviergauthier are you seeing this? Last year I created a fork of this project because I deemed it to be "inactive". Then suddenly the project became active again, but now it seems to be stalled once more.

If you are having troubles to support this project on your own I'm willing to help out where I can.
